web/tests/filter.test.cjs

```diff
@@ -556,6 +566,10 @@ test("basics", () => {
 
     assert.deepEqual(filter.terms(), [...terms, {operator: "with", operand: "12"}]);
     assert.equal(filter.has_exactly_channel_topic_operators(), false);
+
+    terms = [{operator: "channel", operand: "foo", negated: false}];
+    filter = new Filter(terms);
+    assert.ok(filter.is_channel_view());
```

Comment:
The way these tests are supposed to work, this should assert all the statements for each set of terms that we bother define.

Reply:
Okay, added asserts for each of the terms for both `is_channel_view` and `has_exactly_channel_topic_operators` (in a new commit).